### PR TITLE
Adding how to guide for liftover

### PR DESCRIPTION
**Pull request by konradjk**
@jigold - the wheel picked you, it must have known...

### CODE REVIEW

File: hail/python/hail/docs/guides/genetics.rst [hunk 1]
```diff
@@ -35,6 +35,38 @@ Convert variants in string format to separate locus and allele fields
...
+:**tags**: liftover
+
+:**description**: Liftover a Table or MatrixTable from one reference genome to another.
+
+:**code**:
+
+    First, we need to set up the two reference genomes (source and destination):
+        >>> rg37 = hl.get_reference('GRCh37')  # doctest: +SKIP
+        >>> rg38 = hl.get_reference('GRCh38')  # doctest: +SKIP
+        >>> rg37.add_liftover('gs://hail-common/references/grch37_to_grch38.over.chain.gz', ref_38)  # doctest: +SKIP
```
Comment (jigold) [inline hail/python/hail/docs/guides/genetics.rst]: The second reference is `rg38`

Comment (konradjk) [reply hail/python/hail/docs/guides/genetics.rst]: whoops, bad copypasta. thanks

**Comment by konradjk** [comment]
not going to dismiss review until i link from the hl.liftover docs back to here, but addressed the one comment for now

**Comment by jigold** [comment]
I was hoping to see the docs preview to make sure everything is rendered correctly, but I keep getting pending status every time I look.

**Comment by tpoterba** [comment]
we need to store past statuses in the CI UI, I think

**Comment by konradjk** [comment]
In case it disappears again, I got one: https://storage.googleapis.com/hail-ci-0-1/ci/21749db8d218eb071103d1266faf3d7393800f57/c1de8ee30be0ce06ac12a78ec0d3067c299ca8f4/index.html

File: hail/python/hail/docs/guides/genetics.rst [hunk 2]
```diff
@@ -35,6 +35,41 @@ Convert variants in string format to separate locus and allele fields
...
+    Note that this approach does not retain the old locus, nor does it verify
+    that the allele has not changed strand. We can keep the old one for
+    reference and filter out any liftover that changed strands using:
+
+    >>> ht = ht.annotate(new_locus=hl.liftover(ht.locus, 'GRCh38', include_strand=True),
+    ...                  old_locus=ht.locus)  # doctest: +SKIP
+    >>> ht = ht.filter(hl.is_defined(ht.new_locus) & ~ht.new_locus.is_negative_strand)  # doctest: +SKIP
+    >>> ht = ht.key_by(locus=ht.new_locus.result)  # doctest: +SKIP
+
+:**dependencies**: :func:`.liftover`, :meth:`.add_liftover`, :meth:`.get_reference`
```
Comment (jigold) [inline hail/python/hail/docs/guides/genetics.rst]: the get_reference link is broken. Maybe it's just `:func:` instead of `:meth:`

**Pull request merged**